documentation/src/main/asciidoc/mapping.asciidoc

```diff
@@ -1475,22 +1473,36 @@ public class Meeting {
     // ...
 ----
 ====
-
-You can also choose to encode the date as string using the `encoding=EncodingType.STRING` of
-`DateBridge`. In this case the dates are stored in the format _yyyyMMddHHmmssSSS_ (using GMT time).
-
++
 [IMPORTANT]
 ====
 A Date whose resolution is lower than `MILLISECOND` cannot be a `@DocumentId`
```

Comment:
I can see why we would enforce this: truncating IDs in any way is bad idea.
But no, this doesn't seem to be enforced anywhere: the only places we call `org.hibernate.search.annotations.DateBridge.resolution()` from are the date/calendar bridge providers, and we don't  check anything about the ID.

Maybe I should simply remove this warning... ? After all, you can also shoot yourself in the foot by assigning any custom field bridge to the document ID.